cesium_app/handlers/model.py

```diff
@@ -143,19 +133,20 @@ def get(self, model_id=None):
         model_path = pjoin(self.cfg['paths:models_folder'],
                            '{}_model.pkl'.format(uuid.uuid4()))
 
-        model_file = File.create(uri=model_path)
-        model = Model.create(name=model_name, file=model_file,
-                             featureset=fset, project=fset.project,
-                             params=model_params, type=model_type)
+        model = Model(name=model_name, file_uri=model_path,
+                      featureset=fset, project=fset.project,
+                      params=model_params, type=model_type)
+        DBSession().add(model)
 
         client = await self._get_client()
 
         model_stats_future = client.submit(
-            _build_model_compute_statistics, fset.file.uri, model_type,
+            _build_model_compute_statistics, fset.file_uri, model_type,
             model_params, params_to_optimize, model_path)
 
         model.task_id = model_stats_future.key
-        model.save()
+        DBSession().commit()
+        DBSession().expire_all()
```

Comment:
I forget why that was there but it looks like it isn't needed anymore. There was some issue previously w/ a model object being changed inside the callback or something like that, and the session detected an incongruity and freaked out.